network/server_test.go

```diff
@@ -724,22 +724,23 @@ func TestTemporaryDial(t *testing.T) {
 		},
 	}
 	servers, err := createServers(2, map[int]*CreateServerParams{0: defaultConfig, 1: defaultConfig})
-
-	assert.NoError(t, err)
+	if err != nil {
+		t.Fatalf("Unable to create servers, %v", err)
+	}
 
 	t.Cleanup(func() {
 		closeTestServers(t, servers)
 	})
-	//add server1 to server0 temperary dial list
-	servers[0].temporaryDials.Store(servers[1].host.ID(), true)
-	//connect server0 to server1
-	err = servers[0].host.Connect(context.Background(), *servers[1].AddrInfo())
+
+	//make a temporary dail to server1
+	err = dialServer(servers[0], *servers[1].AddrInfo(), true)
```

Comment:
You can inline this error

Reply:
Done, commit-[link](https://github.com/0xPolygon/polygon-edge/pull/343/commits/9f5c60e3ab5d0828f364ee2b5165d6fe36b62bf1)